ckanext/canada/tables/aistrategy.yaml

```diff
@@ -0,0 +1,818 @@
+    excel_error_formula: 'OR({default_formula},LEN({cell})-LEN(SUBSTITUTE({cell},",",""))>0)'
+    validation:
+      en: |
+        This field must not be empty.
+        This field cannot contain commas.
+      fr: |
+        Ce champ ne doit pas être vide.
+        Ce champ ne peut pas contenir de virgules.
+
+  # DEP 1-2
+  - datastore_id: reporting_period
+    label:
+      en: Reporting Period
+      fr: Période de déclaration
+    description:
+      en: Year and month of the update
+      fr: Année et mois de la mise à jour
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Controlled List
+    datastore_type: text
+    validation:
+      en: Must match expected completion options in the controlled list.
+      fr: Doit correspondre aux options d'achèvement prévues de la liste contrôlée.
+    occurrence: Single
+    excel_full_text_choices: true
+    choices:
+      2025-Q3:
+        en: 2025-Q3
+        fr: 2025-T3
+      2025-Q4:
+        en: 2025-Q4
+        fr: 2025-T4
+      2026-Q1:
+        en: 2026-Q1
+        fr: 2026-T1
+      2026-Q2:
+        en: 2026-Q2
+        fr: 2026-T2
+      2026-Q3:
+        en: 2026-Q3
+        fr: 2026-T3
+      2026-Q4:
+        en: 2026-Q4
+        fr: 2026-T4
+      2027-Q1:
+        en: 2027-Q1
+        fr: 2027-T1
+      2027-Q2:
+        en: 2027-Q2
+        fr: 2027-T2
+
+  # DEP 1-3
+  - datastore_id: priority_area
+    label:
+      en: Priority Area
+      fr: Domaine Prioritaire
+    description:
+      en: The strategic focus area within the AI Strategy Implementation
+      fr: Le domaine d'action stratégique dans le cadre de la mise en œuvre de la stratégie en matière d'IA
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Controlled List
+    datastore_type: text
+    validation:
+      en: Must match one of the values in the controlled list.
+      fr: Doit correspondre à l'une des valeurs de la liste contrôlée.
+    occurrence: Single
+    excel_full_text_choices: true
+    choices:
+      A:
+        en: Central AI Capacity
+        fr: Capacité centrale d’IA
+      B:
+        en: Policy, Legislation, and Governance
+        fr: Politiques, lois et gouvernance
+      C:
+        en: Talent and Training
+        fr: Talents et formation
+      D:
+        en: Engagement, Transparency, and Value to Canadians
+        fr: Engagement, transparence et valeur pour les Canadiens
+
+  # DEP 1-4
+  - datastore_id: key_action
+    label:
+      en: Key Action
+      fr: Action Clé
+    description:
+      en: The key action described in the AI Strategy
+      fr: L'action clé décrite dans la stratégie en matière d'IA
+    obligation: Mandatory
+    excel_required: true
+    form_required: true
+    format_type: Controlled List
+    datastore_type: text
+    validation:
+      en: Must match predefined action statements.
+      fr: Doit correspondre à des déclarations d'action prédéfinies.
```

Comment:
adapt the nap5 excel_choice_range_formula and excel_error_formula values for this field so that the drop-downs are hierarchical and errors correctly displayed

Reply:
(same for sub-action and activity)

Reply:
also add:
```yaml
form_attrs:
  cascading_select_field: priority_area
```
and similar for sub-action and activity